DeviceTests/DeviceTests.Shared/Battery_Tests.cs

```diff
@@ -1,46 +1,36 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using Xamarin.Essentials;
+﻿using Xamarin.Essentials;
 using Xunit;
 
 namespace DeviceTests
 {
+    // TEST NOTES:
+    //   - the battery is not emulated on iOS simulators
     public class Battery_Tests
     {
         [Fact]
+#if __IOS__
         [Trait(Traits.DeviceType, Traits.DeviceTypes.Physical)]
+#endif
```

Comment:
This `[Trait]` and `#if` combo replaces the `DeviceInfo.DeviceType == DeviceType.Virtual && DeviceInfo.Platform == DeviceInfo.Platforms.iOS` condition.